cobalt/loader/image/animated_image_tracker.cc

```diff
@@ -140,13 +141,27 @@ void AnimatedImageTracker::Reset() {
 }
 
 void AnimatedImageTracker::OnRenderTreeRasterized() {
+  bool is_main_webmodule = name_ == "MainWebModule";
   count_animated_images_active = playing_urls_.size();
+  if (is_main_webmodule) {
+    UmaHistogramExactLinear("AnimatedImage.MainWebModule.Active",
+                            count_animated_images_active, UINT_MAX);
+  }
   for (const auto& playing_url : playing_urls_) {
```

Comment:
the `playing_urls` for loop below ? It's usually 0 to 1 images, maybe a few at most, because we can't decode more. The request is to be able to push it to ten - see linked bugs in original PR.

Reply:
Ack, sounds like the main driver here is the wrapping function itself which is called very often. Histograms macros would be your best bet here.